src/Miki/Modules/Accounts/AccountsModule.cs

```diff
@@ -1191,14 +1193,17 @@ public async Task DailyStatsAsync(IContext e)
                 var dailyService = e.GetService<IDailyService>();
                 var daily = await dailyService.GetOrCreateDailyAsync((long)e.GetAuthor().Id);
 
-                var time = (daily.LastClaimTime.AddHours(23) - DateTime.UtcNow).ToTimeString(e.GetLocale());
-                var embed = new EmbedBuilder()
-                    .SetTitle(e.GetLocale().GetString("daily_stats_title", $"{e.GetAuthor().Username}"))
-                    .SetColor(253, 216, 136);
+                var locale = e.GetLocale();
+                var timeLeft = daily.LastClaimTime.AddHours(23) - DateTime.UtcNow;
+                var timeString = timeLeft.Seconds > 0 ? timeLeft.ToTimeString(locale) : "Now!";
 
-                embed.AddInlineField(e.GetLocale().GetString("daily_stats_current_streak"), $"{daily.CurrentStreak}");
-                embed.AddInlineField(e.GetLocale().GetString("daily_stats_longest_streak"), $"{(daily.LongestStreak >= daily.CurrentStreak ? daily.LongestStreak:daily.CurrentStreak)}");
-                embed.AddInlineField(e.GetLocale().GetString("daily_stats_time"), $"{time}");
+                var embed = new EmbedBuilder()
+                    .SetTitle(locale.GetString("daily_stats_title", $"{e.GetAuthor().Username}"))
+                    .SetColor(253, 216, 136)
+                    .AddInlineField(locale.GetString("daily_stats_current_streak"), $"{daily.CurrentStreak}")
+                    .AddInlineField(locale.GetString("daily_stats_longest_streak"), 
+                        $"{(daily.LongestStreak >= daily.CurrentStreak ? daily.LongestStreak:daily.CurrentStreak)}")
```

Comment:
This already happens in the service layer. Seems odd to do it twice

Reply:
Yeah, I left it as an edge case incase the current streak gets changed without the longest streak being changed as well

Reply:
But this never happens, right?